client/src/app/components/template/template.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```

Comment:
I would remove this to have all imports grouped in a block - just being picky :)